frontend/lib/Repository/AdminRepository/admin_gateway.dart

```diff
@@ -11,4 +13,14 @@ class AdminGateway {
         restRepository.getGroupsSummary();
     return summary;
   }
+
+  Future<bool> deleteGroup(int id) {
+    final Future<bool> response = restRepository.deleteGroup(id);
+    return response;
+  }
```

Comment:
Myślę, że w przypadku takich krótkich funkcji dobrze by było korzystać z tego zapisu, powienieważ jest krótszy (oszczędzamy 3 linijki) i wygodniejszy w czytaniu 
```suggestion
  Future<bool> deleteGroup(int id) => restRepository.deleteGroup(id);
```